io_scene_godot/__init__.py

```diff
@@ -75,43 +75,51 @@ class ExportGodot(bpy.types.Operator, ExportHelper):
     use_export_shape_key = BoolProperty(
         name="Export Shape Key",
         description="Export all the shape keys in mesh objects",
-        default=True,
+        default=False,
     )
     use_export_selected = BoolProperty(
         name="Selected Objects",
         description="Export only selected objects (and visible in active "
                     "layers if that applies).",
         default=False,
-        )
+    )
     use_exclude_ctrl_bone = BoolProperty(
         name="Exclude Control Bones",
         description="Do not export control bones (bone.use_deform = false)",
         default=True,
     )
     use_export_animation = BoolProperty(
         name="Export Animation",
-        description="Export all the animation actions (include those "
-                    "in nla_tracks), notice if an animated object has "
-                    "an ancestor also has animated, its animation would "
-                    "go into the ancetor's AnimationPlayer",
+        description="Export all the animation actions (include actions "
+                    "in nla_tracks), note that by default blender animation "
+                    "is exported as actions, so every node would have their "
+                    "own AnimationPlayer hold their actions",
         default=True,
-        )
-    use_seperate_animation_player = BoolProperty(
-        name="Seperate AnimationPlayer For Each Object",
-        description="Create a seperate AnimationPlayer node for every"
-                    "blender object which has animtion data",
+    )
+    squash_animation_players = BoolProperty(
+        name="Squash actions",
+        description="Animation is exported as actions of nodes, but instead "
+                    "of having an individual AnimationPlayer for each node, "
+                    "this configuration would squash children nodes' actions "
+                    "to their parents",
+        default=False,
+    )
+    scene_animation = BoolProperty(
+        name="Animation as scene animation",
+        description="All the animations of the whole scene would be placed "
+                    "into one AnimationPlayer",
         default=False,
     )
     use_mesh_modifiers = BoolProperty(
         name="Apply Modifiers",
         description="Apply modifiers to mesh objects (on a copy!).",
-        default=True,
-        )
+        default=False,
```

Comment:
Similar as shapekeys. I would leave this enabled by default.

Reply:
ok I will change it back